### PR TITLE
Plugged FITS loading memory leak.

### PR DESCRIPTION
**Pull request by rzellem**
Thanks to @jpl-jengelke for figuring out how to fix this problem!

### CODE REVIEW

File: exotic.py [hunk 1]
```diff
@@ -972,11 +972,15 @@ def realTimeReduce(i):
...
     for imageFile in timeSortedNames:
 
-        hDul = fits.open(imageFile)  # opens the fits file
+        hDul = fits.open(name=imageFile, memmap=False, cache=False, lazy_load_hdus=False)  # opens the fits file
         # Extracts data from the image file and puts it in a 2D numpy array: imageData
         imageData = fits.getdata(imageFile, ext=0)
         header = fits.getheader(imageFile)
 
+        hdul.close()
+        del hdul[0].data
```
Comment (pearsonkyle) [inline exotic.py:981]: Why no `del(hdul)` to delete the data and the header?

Comment (rzellem) [reply exotic.py]: Good point. Changing to `del hdul` instead.

Comment (jpl-jengelke) [reply exotic.py]: Note the case of the variables. ... `hDul` versus `hdul` in the `del` statement. There won't be a need for `hdul = None` if `del hdul` is used. 

On another note, the same issue is apparent in several other areas in the code. where `fits.open(...)` is utilized.

**Comment by jpl-jengelke** [comment]
Not sure this was published, so reposting it here. ... 

Note the case of the variables. ... `hDul` versus `hdul` in the `del` statement. There won't be a need for `hdul = None` if `del hdul` is used.

On another note, the same issue is apparent in several other areas in the code. where `fits.open(...)` is utilized.

**Comment by rzellem** [comment]
Addressed comments from both @pearsonkyle and @jpl-jengelke in latest push. Please review and approve.

**Comment by rzellem** [comment]
Addressed @jpl-jengelke 's concerns. Using superpowers to merge.

**Comment by rzellem** [comment]
Opening back up again because @rzellem found a bug....

**Pull request merged**